src/constants.zig

```diff
@@ -436,7 +436,8 @@ comptime {
     assert(journal_iops_read_max > 0);
     assert(journal_iops_write_max > 0);
     assert(client_replies_iops_read_max > 0);
-    assert(client_replies_iops_write_max > 0);
+    // Require at least two: one repair, and one commit.
+    assert(client_replies_iops_write_max >= 2);
```

Comment:
I think we no longer need this? Now, even with one IOP it should be fine?

Reply:
Good point!